src/plugins/mini/Examples/read.ini

```diff
@@ -0,0 +1,15 @@
+keyWithoutLeadingWhitespace = 	valueWithLeadingWhiteSpace
```

Comment:
> I moved the files to the main plugin folder.

The installation of the files won't work then.

> using the same name for a subfolder is /pretty useless

We could also rename all of these subfolders to `examples` (please lower case for consistency) and make the installation routine `cmake/Modules/LibAddPlugin.cmake` line 91 rename the configs.

But please in a different PR, for now please name it according the convention.

Please also add `INSTALL_TEST_DATA` so that your test can be executed after Elektra was installed.